neonvm-runner/Dockerfile

```diff
@@ -21,12 +20,16 @@ RUN apk add --no-cache \
     busybox-extras \
     e2fsprogs \
     qemu-system-x86_64 \
+    qemu-system-aarch64 \
```

Comment:
question: What is the expected behavior when installing either of these on the other architecture?

(is it worth only installing one, depending on target arch?)

Reply:
As mentioned in another thread, it was done this way to simplify builds. There is a PR to make proper cross-platform images, which might be extended with changes to install only platform-specific qemu package.